components/automate-gateway/gateway/services.go

```diff
@@ -344,6 +344,8 @@ type ProfileRequest struct {
 }
 
 func (s *Server) ProfileCreateHandler(w http.ResponseWriter, r *http.Request) {
+	// TODO (tc): This ctx is getting overriden by the authRequest call below and is never used.
+	// nolint: ineffectual
 	ctx, cancel := context.WithCancel(r.Context())
```

Comment:
As mentioned in slack, these two lines could go away. It's ok if you don't want to add that to this PR. If you'd want to, that would be cool, too.